js/adslot.js

```diff
@@ -127,6 +135,16 @@ export class AdSlot extends React.Component {
     }
   }
 
+  slotRegisterCallback() {
```

Comment:
I love this!, the past months i've been thinking in a way to let the app code know what the slot-ids of the registered ads are, this is such a great solution.